insights-inventory-client/src/main/java/org/candlepin/insights/inventory/client/InventoryServiceProperties.java

```diff
@@ -32,6 +37,9 @@
     private int apiHostUpdateBatchSize = 50;
     private int staleHostOffsetInDays = 0;
 
+    @DurationUnit(ChronoUnit.HOURS)
+    private Duration hostLastSyncThresholdHours = Duration.ofHours(24);
```

Comment:
I'd prefer to drop units from the name of the property (`hostLastSyncThreshold`). I'd leave of `@DurationUnit` too (we can always use explicit units in config files per [spring docs](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration)).

Also, can we please have the default in only the code (commented out in the config file would be fine).